azure-pipelines.yml

```diff
@@ -1,39 +1,19 @@
 trigger:
 - master
 
-pool:
-  vmImage: 'Ubuntu-16.04'
+jobs :
 
-# We build using a Docker container generated by GafferHQ/build
-container: gafferhq/build:0.0.0
+  - template: config/azure/build.yaml
```

Comment:
Can you explain why we need this template, rather than being able to do everything with a `matrix` in the main file?

Reply:
See notes on https://github.com/GafferHQ/gaffer/pull/3217/commits/f72304d3007cf86fdaa67e531d2e9948e9a86b1e